globals.h

```diff
@@ -95,7 +95,7 @@ WHERE SIG_ATOMIC_VOLATILE_T SigWinch;
 
 WHERE int CurrentMenu;
 
-WHERE struct Alias *Aliases;
+WHERE TAILQ_HEAD(AliasList, Alias) Aliases INITVAL(TAILQ_HEAD_INITIALIZER(Aliases));
```

Comment:
Can you split this up, please.
`mutt/list.h` defines both the ListNode (Alias) and ListHead (AliasList)